### PR TITLE
fix make_iap_request

### PR DESCRIPTION
**Pull request by 3a3**
Fixes https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3732

### CODE REVIEW

**Comment by kurtisvg** [comment]
@leahecole - FYI

**Comment by leahecole** [comment]
I need to double check it in the context of the tutorial

File: functions/composer/composer_storage_trigger.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 import google.oauth2.credentials
 import google.oauth2.service_account
 import requests
-
+from google.oauth2 import id_token
```
Comment (arithmetic1728) [inline functions/composer/composer_storage_trigger.py:24]: I think this import line needs to be put right after line 20, and there should be 2 empty lines between `import requests` and `IAM_SCOPE=...`

Comment (3a3) [reply functions/composer/composer_storage_trigger.py]: Updated

**Pull request merged**